qiskit/transpiler/passes/basis/basis_translator.py

```diff
@@ -597,97 +594,3 @@ def _basis_search(equiv_lib, source_basis, target_basis):
         graph.remove_node(dummy)
 
     return rtn
-
-
-def _compose_transforms(basis_transforms, source_basis, source_dag):
-    """Compose a set of basis transforms into a set of replacements.
-
-    Args:
-        basis_transforms (List[Tuple[gate_name, params, equiv]]): List of
-            transforms to compose.
-        source_basis (Set[Tuple[gate_name: str, gate_num_qubits: int]]): Names
-            of gates which need to be translated.
-        source_dag (DAGCircuit): DAG with example gates from source_basis.
-            (Used to determine num_params for gate in source_basis.)
-
-    Returns:
-        Dict[gate_name, Tuple(params, dag)]: Dictionary mapping between each gate
-            in source_basis and a DAGCircuit instance to replace it. Gates in
-            source_basis but not affected by basis_transforms will be included
-            as a key mapping to itself.
-    """
-    example_gates = _get_example_gates(source_dag)
-    mapped_instrs = {}
-
-    for gate_name, gate_num_qubits in source_basis:
-        # Need to grab a gate instance to find num_qubits and num_params.
-        # Can be removed following https://github.com/Qiskit/qiskit-terra/pull/3947 .
-        example_gate = example_gates[gate_name, gate_num_qubits]
-        num_params = len(example_gate.params)
-
-        placeholder_params = ParameterVector(gate_name, num_params)
-        placeholder_gate = Gate(gate_name, gate_num_qubits, list(placeholder_params))
-        placeholder_gate.params = list(placeholder_params)
-
-        dag = DAGCircuit()
-        qr = QuantumRegister(gate_num_qubits)
-        dag.add_qreg(qr)
-        dag.apply_operation_back(placeholder_gate, qr, (), check=False)
-        mapped_instrs[gate_name, gate_num_qubits] = placeholder_params, dag
-
-    for gate_name, gate_num_qubits, equiv_params, equiv in basis_transforms:
```

Comment:
It looks like the documentation is wrong then for `basis_transforms` in the original Python code, no? Since the tuple does not have the number of bits there.